ui/src/app/cluster-workflow-templates/cluster-workflow-template-details.tsx

```diff
@@ -63,7 +68,11 @@ export function ClusterWorkflowTemplateDetails({history, location, match}: Route
     useEffect(() => {
         (async () => {
             try {
+                const workflowList = await services.workflows.list('', null, [`${models.labels.clusterWorkflowTemplate}=${name}`], {limit: 50});
                 const info = await services.info.getInfo();
+
+                setWorkflows(workflowList.items);
+                setColumns(info.columns);
```

Comment:
good re-use here. note that there is an issue opened to refactor the `info` request into context so that it isn't called on every page of the SPA: #12771